charts/kubescape-operator/templates/gateway/deployment.yaml

```diff
@@ -25,8 +24,7 @@ spec:
     type: RollingUpdate
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ .Values.gateway.name }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
@smalinet it's not clear to me whether we want to remove `app.kubernetes.io/instance` from all `matchLabels` - what's the best practice?

Reply:
@matthyx These keys are dynamically generated by the `selectorLabels` in the __helpers.tpl_ file.
This is a first iteration but in my opinion the good approach would be something like:

```yaml
    matchLabels:
      {{- include "kubescape-operator.selectorLabels" . | nindent 6 }}
      app.kubernetes.io/component: {{ .Values.gateway.name }}
```